package.json

```diff
@@ -56,6 +56,8 @@
     "coverage": "hardhat coverage",
     "docgen": "hardhat docgen",
     "topup": "hardhat run scripts/top-up.js --network localhost",
+    "adapter_add": "hardhat run scripts/test-adapter-add.js --network localhost",
```

Comment:
rename to test_adapter_add and test_adapter_remove

Reply:
Done